resources/new-contract/csharp/dotnet-tools.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "neo.express": {
+      "version": "2.0.37-preview",
```

Comment:
Is there a way to share the express version between the template contract and the version used in DevTracker?